CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.30.0 Release
+- Release Date: April 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
```

Comment:
Note to self: remember to change in app stores.

Reply:
@amyblais @enahum why are we changing min server requirements in v1.30?

Reply:
@esethna Because we don't support v5.9 ESR anymore as of April 15th.

Reply:
Are there any changes in this release that will break when used with server version v5.18 or lower?

Reply:
@amyblais have we communicated this with customers?

> Because we don't support v5.9 ESR anymore as of April 15th.

Reply:
There should not be anything in this release that is breaking compatibility, but I'm with Amy here, wee should not say that we support a version previous to the ESR. We've been through this path before.

Reply:
@amyblais can you help confirm with QA team that the app was tested on v5.9 and passed? I'm concerned that since we did not give customers much of a heads up about the ESR going out support that many have not upgraded, and we should verify that we don't have issues with the app on the old ESR. But I agree we should update the doc and app stores to the latest ESR

Reply:
@esethna Yes it was tested on v5.9.